Mathlib/MeasureTheory/Measure/NullMeasurable.lean

```diff
@@ -515,4 +515,56 @@ end Measure
 
 end IsComplete
 
+section indicator
```

Comment:
Is there a better place for these analogues of `Mathlib.Topology.IndicatorConstPointwise`? They are (now) needed in `Mathlib.MeasureTheory.Constructions.BorelSpace.Metrizable` and `Mathlib.MeasureTheory.Integral.Lebesgue` (they have for example `Mathlib.MeasureTheory.MeasureSpace` as a common ancestor, but this file or a completely new one seem more appropriate places).

Reply:
Maybe these lemmas are not needed anymore if you prove `tendsto_indicator_const_iff_forall_eventually` for a single `x` (which is a reformulation of `tendsto_ite` mentioned elsewhere), because then you can just `simp_rw` with that lemma?